dev/TabView/TabView.xaml

```diff
@@ -75,74 +90,23 @@
-                                                                Style="{StaticResource RepeatButtonRevealStyle}"
-                                                                Visibility="Collapsed"
-                                                                Content="&#xE76B;"/>
-
-                                                            <ScrollContentPresenter x:Name="ScrollContentPresenter"
-                                                                Grid.Column="1"
-                                                                Padding="{TemplateBinding Padding}"
-                                                                contract4Present:TabFocusNavigation="Once" />
-
-                                                            <RepeatButton x:Name="ScrollIncreaseButton"
-                                                                Grid.Column="2"
-                                                                VerticalAlignment="Bottom"
-                                                                HorizontalAlignment="Center"
-                                                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
-                                                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
-                                                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
-                                                                Background="Transparent"
-                                                                BorderThickness="1"
-                                                                Delay="50"
-                                                                FontFamily="Segoe MDL2 Assets"
-                                                                Interval="100"
-                                                                Style="{StaticResource RepeatButtonRevealStyle}"
-                                                                Visibility="Collapsed"
-                                                                Content="&#xE76C;"/>
-                                                        </Grid>
-                                                    </Border>
-                                                </ControlTemplate>
-                                            </Setter.Value>
-                                        </Setter>
-                                    </Style>
-                                </ScrollViewer.Style>
+                                Style="{StaticResource TabScrollViewerStyle}">
 
                                 <ItemsPresenter x:Name="TabsItemsPresenter"
                                     Padding="{TemplateBinding Padding}" />
 
                             </ScrollViewer>
 
-                            <ContentPresenter
+                            <Button
```

Comment:
>Button [](start = 29, length = 6)

Should this button have a default Ctrl+T KeyboardAccelerator, or some way for the control to enable that?  It may also make sense to allow the app to set a custom KA for this button depending on their keyboarding model.

Reply:
No, we have explicitly decided against that.

Reply:
Why?  In Edge, if I hover over the + button I see "New Tab (Ctrl+T)" which feels like a good thing to at least have as an option.

Reply:
We decided not to do any keyboard shortcuts. PMs all argued about it, that was the decision. There was concern it might cause problems if an existing app adopted the control but had already bound ctrl-T to some other thing in their app.

Reply:
We had a similar concern with RichEditBox, so it has an API to disable accelerators that you don't want (like Ctrl+I for italics).  I could imagine an API of two shapes for this:

1) IsNewTabKeyboardAcceleratorEnabled
2) NewTabKeyboardAccelerators collection that the app can set, which you forward on to the button